how-to-docs/certificates.md

```diff
@@ -145,9 +145,9 @@ spec:
 
 **HTTP Challenge**
 
-The above example requires that the DNS zone of the ingress hostname you are adding is located in the AWS account that kube-cert-manager resides. Assuming this is not the case you will need to use a HTTP challenge rather than the default DNS one. You wil need to change your DNS name to a CNAME of ingress-external / ingress-internal ENVIRONMENT.acp.homeoffice.gov.uk i.e assuming notprod environment mysite.domain.com -> ingress-external.notprod.acp.homeoffice.gov.uk
+The above example requires that the DNS zone of the ingress hostname you are adding is located in the AWS account that kube-cert-manager resides, i.e. it has the ability to update records on your behalf. Assuming this is not the case you will need to use a HTTP challenge rather than the default DNS one. You will need to change your DNS hostname to a CNAME of ingress-external / ingress-internal ENVIRONMENT.acp.homeoffice.gov.uk i.e assuming notprod environment mysite.domain.com -> ingress-external.notprod.acp.homeoffice.gov.uk
 
-#### **Note: the HOSTED domains are linked to the cluster i.e. notprod.acp.homeoffice.gov.uk is in notprod and prod is prod.acp.homeoffice.gov.uk etc etc. So if the domain you are requesting for in your ingress IS NOT the specific cluster domain YOU NEED to use the http challenge and update the DNS first to point our external ingress by default DNS points to the internal ingress**
+#### **Note: if you are placing your ingress behind the VPN i.e. using the ingress-class `ingress-internal`, the HTTP challenge will not work, please contact the ACP team as you will need to move or delegate you domain to Route53**
```

Comment:
`you domain to Route53` -> `your domain to Route53`

Maybe for a different PR, but we should provide a set of ingress examples for the different scenarios (internal domain, external domain, managed / not managed by Route53 etc).

Reply:
Agree - I'm confused too. There is such a ticket btw but I've been told to wait till the ingress placeholder ticket's decided on - https://github.com/UKHomeOffice/application-container-platform-board/issues/969